marklogic-sesame/src/test/java/com/marklogic/semantics/sesame/MarkLogicRepositoryConnectionTest.java

```diff
@@ -515,7 +515,7 @@ public void testAddStatements() throws Exception{
         BooleanQuery booleanAliceQuery = conn.prepareBooleanQuery(QueryLanguage.SPARQL, checkAliceQuery);
         Assert.assertTrue(booleanAliceQuery.evaluate());
 
-        String checkBobQuery = "ASK { <http://example.org/people/bob> <http://example.org/ontology/age> '123123123123'^^<http://www.w3.org/2001/XMLSchema#double> .}";
+        String checkBobQuery = "ASK { <http://example.org/people/bob> <http://example.org/ontology/age> '1.23123123123E11'^^<http://www.w3.org/2001/XMLSchema#double> .}";
```

Comment:
I could not find any official statement on a behaviour change but doing the change made the "ask" work for me.